pyproject.toml

```diff
@@ -36,6 +36,9 @@ MAPPED = "true"
 CONDITION = "WHERE a.geom IS NOT NULL"
 GEOM = "geom_2263"
 
+[tool.sqlfluff.templater.jinja]
+apply_dbt_builtins = true
+
```

Comment:
has no effect until we start linting queries written for dbt with jinja templating